### ISSUE TITLE
Is there a way vimwiki just provide core functionalities while leave the syntax and folding to other plugins?

### ISSUE THREAD

**Issue by mizhozan** [vimwiki/vimwiki#1308]
I have been using vim-markdown for my markdown syntax highlighting and folding. However, now I'm using vimwiki to handle my notes. The problem is, vimwiki do not fully support markdown syntax (at least pandoc) and also do not highligh properly. For the case of math `$$` it has its own convention `{{$` which breaks markdown syntax. Plus, the folding also not that useful as it fold the whole H1 level, etc. 

So, my question is, is there a way to disable vimwiki syntax highlighting and let's vim-markdown handle syntax (and even folding), while in the background vimwiki works with links, backlinks, etc. The core functionality of vimwiki for me is its friction-less linking and backlinking.

**tldr; Is there a way vimwiki just provide core functionalities such as <CR> link opening, `:VWB` while leave the syntax and folding to other plugins?**

**Comment by tinmarino** [comment]
Hi @mizhozan, no there is currently not. You must cancel each configuration one by one. For the typeface for example, see #1346.

We must get a more user friendly configuration at some point but this may take year, sorry.

**Comment by tinmarino** [comment]
Closing <= no code expected

**Comment by infinitewhileloop** [comment]
@mizhozan This might be what you are looking for: https://github.com/lervag/wiki.vim
